docs/modules/ROOT/pages/mgmt_services/route_manager.adoc

```diff
@@ -37,24 +37,24 @@ image::route_manager_service_architecture.PNG[]
 
 === Key Workflow
 
-** For case #4, add or delete a non-gateway port.
+==== For case #4, add or delete a non-gateway port.
```

Comment:
@cj-chung A side comment: Can we add a link in the high-level routing_design doc pointing to this doc, which includes customer-facing APIs not included in the high-level doc?

Thanks,

https://github.com/cj-chung/alcor/blob/routemanager/docs/modules/ROOT/pages/high_level/routing_design.adoc